src/nft/pages/asset/Asset.tsx

```diff
@@ -16,7 +16,17 @@ const AssetContainer = styled.div`
   width: 100%;
   justify-content: center;
   gap: 60px;
-  padding: 48px 40px 0 40px;
+  padding: 48px 48px 0 48px;
+
+  @media (max-width: 960px) {
```

Comment:
can you use the predefined BREAKPOINT constants here pls for consistency: code/interface/src/components/Tokens/constants.ts 

although we should probably also move these constants to a different shared location, not just under Tokens🤔

Reply:
I checked just now and the exact sizings are different, not sure why the design doesn't match our usual padding